generated/react-native-flexible-sync/package.json

```diff
@@ -7,22 +7,27 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "prettier": "prettier --write src/*",
+    "generate:prod-partition-based-sync": "npx bluehawk copy . --output=../generated/react-native-partition-based-sync --ignore=realm.json --state=partition-based-sync",
+    "generate:prod-flexible-sync": "npx bluehawk copy . --output=../generated/react-native-flexible-sync --ignore=realm.json --state=flexible-sync",
+    "generate:prod": "npm run generate:prod-partition-based-sync && npm run generate:prod-flexible-sync"
```

Comment:
Good idea to have these bluehawk commands in the package.json, but I suggest we shorten the names:
- generate-both # since it generates both FS and PBS; or alternatively 'generate-all' or just 'generate'
- generate-pbs
- generate-fs